src/test/kotlin/calculator/usecase/SplitParserTest.kt

```diff
@@ -21,7 +21,7 @@ class SplitParserTest {
     @ValueSource(strings = ["//;\\n1;2;3", "//$\\n1$2$3", "//%\\n1%2%3"])
     @ParameterizedTest
     fun `1,2,3이 custom separator 로 이루어졌을 때 결과 listOf(1,2,3)`(input: String) {
-        val expected = listOf(1, 2, 3)
+        val expected = listOf(1,2,3)
```

Comment:
```suggestion
        val expected = listOf(1, 2, 3)
```
ktlint에서 이 부분 때문에 통과하지 못하고 있습니다!
커밋 시 ktlint hook이 안걸려있으시다면 걸어주세요! 🙏